go/client/simplefs_windows.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build windows
+
+package client
+
+import (
+	"context"
+	"errors"
+	"path/filepath"
+	"strings"
+
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func doSimpleFSRemoteGlob(g *libkb.GlobalContext, ctx context.Context, cli SimpleFSInterface, path keybase1.Path) ([]keybase1.Path, error) {
+
+	var returnPaths []keybase1.Path
+	directory := filepath.ToSlash(filepath.Dir(path.Kbfs()))
+	base := filepath.Base(path.Kbfs())
+
+	// We know the filename has wildcards at this point.
+	// kbfs list only works on directories, so build a glob from a list result.
+
+	g.Log.Debug("doSimpleFSRemoteGlob %s", path.Kbfs())
+
+	if strings.ContainsAny(filepath.Base(directory), "?*[]") == true {
```

Comment:
But maybe we'll be doing weird things in that case no matter what (double interpreting globs vs failing on literal stars), so maybe I'm asking the impossible.